### PR TITLE
Getting alfons works with vs 2015.

### PR DESCRIPTION
**Pull request by lygstate**

### CODE REVIEW

**Comment by karimnaaji** [comment]
Related to https://github.com/tangrams/tangram-es/pull/836

File: deps/harfbuzz-icu-freetype [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 364b30bb26f3c4821e675f6b3027b4af74340ae4
+Subproject commit 1fe3240ba2b89a8cab9d7c4c586ecfbf66f18f03
```
Comment (tallytalwar) [inline deps/harfbuzz-icu-freetype:1]: This got merged in the original project branch (https://github.com/tangrams/harfbuzz-icu-freetype/commits/remove-unused-deps), can you update this to: eccc706464b7b9858f44ffa779f3ba508ab648a7

Will merge this PR once this is done. Thanks

Comment (karimnaaji) [inline deps/harfbuzz-icu-freetype:1]: Also, the active target of this PR should be https://github.com/hjanetzek/alfons/tree/wip (the branch we currently use in tangram-es).

**Comment by karimnaaji** [comment]
Merged as https://github.com/hjanetzek/alfons/commit/6284c61ab41d77e5b33e6246187641243fc0a5d2 on `wip` branch. Closing this.

**Pull request closed without merging**